jenkins-scripts/dsl/gazebo_ros_pkgs.dsl

```diff
@@ -22,27 +22,62 @@ extra_gazebo_versions = [ 'kinetic' :  ['8','9'],
 
 bloom_debbuild_jobs = [ 'gazebo-dev', 'gazebo-msgs', 'gazebo-plugins', 'gazebo-ros', 'gazebo-ros-control', 'gazebo-ros-pkgs' ]
 
-Job create_common_compilation(String job_name,
-                              String ubuntu_distro,
-                              String ros_branch,
-                              String gz_version,
-                              String script_name)
+String get_branch_from_rosdistro(ros_distro) {
+  if (ros_distro == current_ros2_branch)
+    return "ros2"
+
+  return "${ros_distro}-devel"
```

Comment:
This needs special treatment for `ros2` release branches, see [this failure](https://build.osrfoundation.org/job/ros2_gazebo_pkgs-ci-default_eloquent-bionic-amd64/10/console):

```
git rev-parse eloquent-devel^{commit} # timeout=10
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
```

Reply:
ouch fixed in b75f605